ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml

```diff
@@ -49,3 +49,52 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit-test,?($|\s.*)
+  - agent: kubernetes
```

Comment:
Not required. e2e-tests are not available in the release-4.10 branch.

Reply:
Updated